index.html

```diff
@@ -40,4 +40,234 @@
+          <li class="tagj"> <a class="html" href="">JavaScript</a></li>
+        </ul>
+        
+        <div class="action">
+         <button class="normal" type="button">See Project</button>
+        </div>
+        
+      </div>
+         
+        
+      
+    </div>
+    
+
+  
+
+  
+    <div class="img-holder">
+      <img class="about-my-self" src="Microverse-Student-Project-1/img6.png" alt="">
+    </div>
+      <div class="top">
+        <h2 class="first-text">About <br><span>Myself</span></h2>
+        <p class="second-text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <div class="social">
+          <a class="buttom-btn" href="#">LET’S CONNECT</a>
+          <ul>
+            <li><a href=""><i class="emo emoji fa-brands fa-github"></i></a></li>
+            <li><a href=""><i class="emo emoji fa-brands fa-linkedin"></i></a></li>
+            <li><a href=""><i class="emo emoji fa-solid fa-hand-peace"></i></a></li>
+            <li><a href=""><i class="emo emoji emoji1 fa-brands fa-twitter"></i></a></li>
+            <li><a href=""><i class="emo emoji fa-solid fa-m"></i></a></li>
+          </ul>
+          <div class="baction">
+            <button class="f-normal" type="button">Get my resume</button>
+          </div>
+        </div>
+      </div>
+      <div class="skill">
+        <div class="frame12">
+          <div class="frame7">
+            <ul>
+              <li><i class="normalization-btn fa-solid fa-chevron-down"></i></li>
+              <li><h2 class="lan">Languages</h2></li>
+            </ul>
+          </div>
+          <div class="language-list">
+            <ul class="items-list">
+              <li><div class="frame9">
+                <img class="javaP" src="Microverse-Student-Project-1/JS.png" alt="JS">
+                <h2 class="javaS">JavaScript</h2>
+              </div></li>
+              <li><div class="frame9">
+                <img class="javaP" src="Microverse-Student-Project-1/html.png" alt="JS">
+                <h2 class="javaS">HTML</h2>
+              </div></li>
+              <li><div class="frame9">
+                <img class="javaP" src="Microverse-Student-Project-1/css.png" alt="JS">
+                <h2 class="javaS">CSS</h2>
+              </div></li>
+            </ul>
+          </div>
+          <div class="frame-8">
+            <ul>
+              <li><i class="normalization-btn fa-solid fa-angle-right"></i></li>
+              <li><h2 class="lan">Frameworks</h2></li>
+            </ul>
+          </div>
+          <div class="frame-8">
+            <ul>
+              <li><i class="normalization-btn fa-solid fa-angle-right"></i></li>
+              <li><h2 class="frame-8">Skills</h2></li>
+            </ul>
+          </div>
+        </div>
+      </div>
```

Comment:
[OPTIONAL ] 
The previous reviewer meant to say that the projects and about myself should each be wrapped in their own `section` tag. Please wrap the about myself in a `section` tag and the same for the projects too 👍